AUTHORS.md

```diff
@@ -1,3 +1,4 @@
+(authors)=
```

Comment:
Why necessary? This file is always referenced with its location, never with the link? (Right?)
```suggestion
```